### ISSUE TITLE
Dell Inspirion - Segmentation Fault

### ISSUE THREAD

**Issue by DellInspirionUser** [libretro/RetroArch#7062]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

Retroarch window crashes to error report on Lubuntu 18.04.
--verbose gives segmentaion error regardless of video driver config settings.

### Expected behavior

Retroarch to load

### Actual behavior

Crash to desktop

`dell@dell-Inspiron-1501:~$ sudo retroarch --verbose

 [INFO] RetroArch 1.7.3 (Git b2ceb50)

 [INFO] === Build =======================================
Capabilities: MMX MMXEXT SSE1 SSE2 SSE3 
Built: May 15 2018

 [INFO] Version: 1.7.3

 [INFO] Git: b2ceb50

 [INFO] =================================================

 [INFO] Environ SET_PIXEL_FORMAT: RGB565.

 [INFO] Version of libretro API: 1

 [INFO] Compiled against API: 1

 [INFO] [Audio]: Set audio input rate to: 29970.03 Hz.

 [INFO] [Video]: Video @ 960x720 error: XDG_RUNTIME_DIR not set in the environment.

 [ERROR] [Wayland]: Failed to connect to Wayland server.
 
 [INFO] [GLX]: GLX_OML_sync_control and GLX_MESA_swap_control supported, using better swap control method...
 
 [INFO] [GL]: Found GL context: x
 
 [INFO] [GL]: Detecting screen resolution 1280x1024.
 
 [INFO] [GLX]: Window manager is Openbox.
 
 [INFO] [GLX]: X = 0, Y = 0, W = 960, H = 720.
 
 [INFO] [GLX]: Found swap function: glXSwapIntervalEXT.
 
 [INFO] [GLX]: glXSwapIntervalEXT(1)

 [INFO] [GL]: Vendor: X.Org R300 Project, Renderer: ATI RS480.
 
 [INFO] [GL]: Version: 2.1 Mesa 18.0.0-rc5.
 
 [INFO] [GL]: Using resolution 960x720
 
 [INFO] [GL]: Default shader backend found: glsl.

 [INFO] [Shader driver]: Using GLSL shader backend.

 [INFO] [GLSL]: Checking GLSL shader support ...
 
 [WARN] [GL]: Stock GLSL shaders will be used.

 [INFO] [GLSL]: Found GLSL vertex shader.

 [INFO] [GLSL]: Found GLSL fragment shader.

 [INFO] [GLSL]: Linking GLSL program.

 [INFO] [GLSL]: Found GLSL vertex shader.

 [INFO] [GLSL]: Found GLSL fragment shader.

 [INFO] [GLSL]: Linking GLSL program.

 [INFO] [GLSL]: Found GLSL vertex shader.

 [INFO] [GLSL]: Found GLSL fragment shader.

 [INFO] [GLSL]: Linking GLSL program.

 [INFO] Setting up menu pipeline shaders for XMB ... 

 [INFO] [GLSL]: Found GLSL vertex shader.

 [INFO] [GLSL]: Found GLSL fragment shader.

 [INFO] [GLSL]: Linking GLSL program.

 [INFO] [GLSL]: Found GLSL vertex shader.

 [INFO] [GLSL]: Found GLSL fragment shader.

 [INFO] [GLSL]: Linking GLSL program.

 [INFO] [GLSL]: Found GLSL vertex shader.

 [INFO] [GLSL]: Found GLSL fragment shader.

 [INFO] [GLSL]: Linking GLSL program.

 [INFO] [GLSL]: Found GLSL vertex shader.

 [INFO] [GLSL]: Found GLSL fragment shader.

 [INFO] [GLSL]: Linking GLSL program.

 [INFO] [GLSL]: Found GLSL vertex shader.

 [INFO] [GLSL]: Found GLSL fragment shader.

 [INFO] [GLSL]: Linking GLSL program.

 [INFO] [GLSL]: Found GLSL vertex shader.

 [INFO] [GLSL]: Found GLSL fragment shader.

 [INFO] [GLSL]: Linking GLSL program.

 [INFO] Resetting shader to defaults ... 

 [INFO] [GL]: Using 4 textures.

 [INFO] [GL]: Loaded 1 program(s).

 [INFO] [Joypad]: Found joypad driver: "udev".

 [INFO] [Font]: Using font rendering backend: freetype.

 [INFO] [X11]: Suspending screensaver (X11, xdg-screensaver).

 [INFO] [Video]: Found display server: x11

 [INFO] [PulseAudio]: Requested 24576 bytes buffer, got 18432.

 [INFO] [Menu]: Found menu display driver: "menu_display_gl".

 [INFO] [Font]: Using font rendering backend: freetype.

 [INFO] [Font]: Using font rendering backend: freetype.

 [INFO] [LED]: LED driver = 'null' 0xbb2200

 [INFO] SRAM will not be saved.

 [INFO] Loading history file: [/home/dell/.config/retroarch/content_history.lpl].

 [INFO] Loading history file: [/home/dell/.config/retroarch/content_favorites.lpl].

 [INFO] Loading history file: [/home/dell/.config/retroarch/content_music_history.lpl].

 [INFO] Loading history file: [/home/dell/.config/retroarch/content_video_history.lpl].

 [INFO] Loading history file: [/home/dell/.config/retroarch/content_image_history.lpl].

 [INFO] [GL]: VSync => on

 [INFO] [GLX]: glXSwapIntervalEXT(1)

 [INFO] [PulseAudio]: Unpausing.

 [INFO] [GL]: VSync => on

 [INFO] [GLX]: glXSwapIntervalEXT(1)

 [INFO] [PulseAudio]: Pausing.
Segmentation fault
dell@dell-Inspiron-1501:~$ 
`

**Comment by parkerlreed** [comment]
Why are you running RetroArch with sudo...

**Comment by hizzlekizzle** [comment]
Yeah, sudo's not recommended. Besides that, that GPU/driver is borderline, with OpenGL 2.1 support only, but it should still work. You could try opening your retroarch.cfg and change the menu_driver from "xmb" to "rgui"...

I don't see any core being loaded there. Does it treat you any better if you try to load a core and content directly from the command line?

**Comment by gouchi** [comment]
@DellInspirionUser Please close this issue, if you don't reproduce the issue with latest release.

You can get Ubuntu package using this [PPA](https://docs.libretro.com/guides/install-gnu/#ubuntu-based) or [Snap package](https://snapcraft.io/retroarch).

If you know how to compile, try

```
git clone https://github.com/libretro/RetroArch.git
cd RetroArch
./configure && make -$(nproc)
```

If you still reproduce the issue, please provide some [debugging logs](https://docs.libretro.com/development/retroarch/debugging/).

Thank you.

**Comment by LibretroAdmin** [comment]
Stale; not reproducible; would need a backtrace
